src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/_params.py

```diff
@@ -25,5 +25,6 @@ def get_subscription_id_list(prefix, **kwargs):  # pylint: disable=unused-argume
 register_cli_argument('logout', 'username', help='account user, if missing, logout the current active account')
 
 register_cli_argument('account', 'subscription', help='Name or ID of subscription.', completer=get_subscription_id_list)
-register_cli_argument('account list', 'all_clouds', options_list=('--all',), help='List all subscriptions across all cloud environments', action='store_true')
+register_cli_argument('account list', 'all_clouds', options_list=('--all-clouds',), help='List all subscriptions across all cloud environments', action='store_true')
```

Comment:
@derekbekoe, because stack profile support is not GA yet. So i figure it is fine to take the shortcut